app/nagbot.py

```diff
@@ -182,6 +236,14 @@ def is_excluded(instance: Instance):
     return False
 
 
+# Some volumes are excluded from terminate actions. These won't show up as recommendations to terminate.
```

Comment:
Learning question: a use case when a volume is excluded from terminate actions.

Reply:
Based on the regex, I would guess that there are certain volumes/instances that have some variant of 'bamboo' in the name that we want to keep running, so these would be excluded from terminate actions

Reply:
I think it'd be ok to remove the bamboo logic (for instances as well), we don't use bamboo anymore